packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -714,6 +738,13 @@ export class PipelineEditor extends React.Component<
       this.handleSavePipeline();
     } else if (action == 'clear') {
       this.handleClearPipeline();
+    } else if (
+      action == 'undo' ||
+      action == 'redo' ||
+      action == 'delete' ||
+      action == 'addComment'
```

Comment:
Added listeners for canvas commands that need to update pipeline-editor buttons. 
`Cut` and `paste` already triggers model update.

Reply:
not an issue but if you wanted you could simplify this to one line
```javascript
} else if (['undo', 'redo', 'delete', 'addComment'].includes(action)) {
```
or if you prefer
```javascript
} else if (['undo', 'redo', 'delete', 'addComment'].indexOf(action) > -1) {
```